libs/npm/core/src/components/grid-table/GridTable.tsx

```diff
@@ -196,8 +196,9 @@ export const GridTable = <T extends object>({
   }, [onChangePage, pageIndex, pageSize, currentPage]);
 
   React.useEffect(() => {
-    onChangeSort(sortBy);
-  }, [onChangeSort, sortBy]);
+    if (sortBy.length !== initialSortBy.length || !sortBy.every((v, i) => v === initialSortBy[i]))
```

Comment:
Not sure if this will work, still need to test.  For some reason react-table is spamming changes to `sortBy` even though nothing changes.  I'm not sure why it does this.